components/inputs/input-number.js

```diff
@@ -275,6 +288,30 @@ class InputNumber extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Lit
 		return [...childErrors, ...errors];
 	}
 
+	_getTooltip() {
+		if (this.disabled) return null;
+		if (this.validationError && this.childErrors.size === 0) {
+			return html`<d2l-tooltip announced for="${this._inputId}" state="error" align="start">${this.validationError}</d2l-tooltip>`;
+		}
+		let lang = '';
```

Comment:
nit: maybe this should just be named hint? Also might be a bit nicer to just default it to null and  just do `if (lang) {` on line 306